seqr/views/apis/saved_variant_api_tests.py

```diff
@@ -330,6 +333,17 @@ def test_saved_variant_data(self):
         self.assertListEqual(variants['SV0000002_1248367227_r0390_100']['familyGuids'], ['F000002_2'])
         self.assertEqual(set(response_json['familiesByGuid'].keys()), {'F000001_1', 'F000002_2', 'F000012_12'})
 
+        # Test empty project
+        empty_project_url = url.replace(PROJECT_GUID, 'R0002_empty')
+        response = self.client.get(empty_project_url)
+        self.assertEqual(response.status_code, 200)
+        empty_response = {k: {} for k in VARIANT_TAG_RESPONSE_KEYS}
+        self.assertDictEqual(response.json(), empty_response)
+
+        response = self.client.get(f'{empty_project_url}?loadProjectTagTypes=true&loadFamilyContext=true')
```

Comment:
this test covers the error, without the change this test returns the same 500 issue

Reply:
just to confirm, by 'empty' do you mean that the project has no saved variants?

Reply:
It has no saved variants or families. Its referred to as the empty project in several test files